### PR TITLE
cluster-setup: Revamp Netmap section

### PR DESCRIPTION
**Pull request by awelzel**
I'm not sure this is too verbose, but given that generally seemed one
needs to find small pieces of info in various places, I'm not feeling too
bad putting a few details in.

This is mostly from a "running without zeekctl" perspective.

### CODE REVIEW

**Comment by ckreibich** [comment]
Thanks Arne, this is great. How about two subsections fro this, one for FreeBSD, one for Linux? The way the section starts out, it sounds like it's completely about FreeBSD.

> This is mostly from a "running without zeekctl" perspective.

Yes, great — we should no longer add material that specifically targets the `zeekctl` scenario.

File: cluster-setup.rst [hunk 1]
```diff
@@ -142,9 +142,88 @@ PF_RING, see the documentation on :ref:`how to configure Zeek with PF_RING
...
+The plugin can also be used on FreeBSD in case libpcap's native support is found
+to be insufficient. The plugin further provides basic zeekctl support.
+
+.. warning::
+
+  If you're using the zeek-netmap plugin on FreeBSD, the interface specification
+  given to Zeek needs to change from `netmap:zeek}0/x` to `netmap::zeek}0/x` - a single colon more.
+  In the first case, Zeek instantiates the libpcap packet source and passes `netmap:zeek}0`
+  as parameter. In the second case, `netmap::` is interpreted by Zeek, it
+  instantiates teh netmap packet source `zeek}0/x` is treated as interfac
```
Comment (ckreibich) [inline cluster-setup.rst:224]: Typos. :-)

**Review by ckreibich — CHANGES_REQUESTED** [review]
Looks great, just requesting changes for the typos and separating FreeBSD and Linux, if you're up for that.

**Comment by awelzel** [comment]
> Looks great, just requesting changes for the typos and separating FreeBSD and Linux, if you're up for that.

Done. It's still fairly FreeBSD heavy, but think this mostly comes from being much more accessible there as compared to Linux. Also not sure we should recommend netmap on Linux..

**Review by ckreibich — APPROVED** [review]

**Pull request merged**